build.gradle.kts

```diff
@@ -27,6 +27,7 @@ dependencies {
     implementation("xpp3:xpp3:1.1.4c")
     implementation("app.revanced:smali:2.5.3-a3836654")
     implementation("app.revanced:multidexlib2:2.5.3-a3836654")
+    // ARSCLib fork with a custom zip implementation to fix performance issues on Android devices
```

Comment:
Why is this not upstream?

Reply:
Possibly a PR that hasn't been merged yet? (and shouldn't block this PR)

Reply:
In that case that should be noted here with a link.

Reply:
It's a PR yeah, but it's been inactive for a month now: https://github.com/REAndroid/ARSCLib/pull/21

I agree that it should be linked.

Reply:
The reason why it is not upstream is because the archive system is being rewritten to also support things like signature spoofing. Archive2 also addresses the performance issues, but is still WIP. We will not be needing the fork after the maintainer bumps the version.
I have added a note about the future obsolescence of the fork.

Reply:
A tracking issue should be made in the fork fo prepare for EOL once upstream has made the changes.

Reply:
> A tracking issue should be made in the fork fo prepare for EOL once upstream has made the changes.

https://github.com/revanced/ARSCLib/issues/2